docs/manual/called-methods-checker.tex

```diff
@@ -6,8 +6,13 @@
 of the form ``call method A before method B''. For the purpose of this
 checker, a method has ``definitely been called'' if it is \emph{invoked}:
 a method that might never return or that might throw an exception has
-a method that might never return or that might throw an exception has
 definitely been called on every path after the call, including exceptional paths.
+The checker also ignores the possibility that an exception might occur as part
+of the call (even if the called method does not throw the exception):
+that is, it assumes that the receiver of every method invocation
+is non-null at each call-site, and that run-time errors (such as \<OutOfMemoryError>
```

Comment:
The assumption is stronger than this.  The checker now assumes that _all_ dereferences (field accesses or method calls) will succeed without throwing an NPE.  Can you clarify?

Reply:
You're right. I think I've made the wording more clear; can you take another look?